projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.ts

```diff
@@ -1,38 +1,41 @@
-import {Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges, ViewEncapsulation} from '@angular/core';
+import {Component, EventEmitter, Input, OnChanges, OnDestroy, OnInit, Output, SimpleChanges, ViewEncapsulation} from '@angular/core';
 import { Annotation } from '../../annotation-set/annotation-view/annotation.model';
 import {select, Store} from '@ngrx/store';
 import * as fromStore from '../../../store/reducers';
 import * as fromTagSelectors from '../../../store/selectors/tags.selectors';
 import * as fromAnnoSelector from '../../../store/selectors/annotations.selectors';
+import {combineLatest, Subscription} from 'rxjs';
 
 @Component({
   selector: 'mv-comment-set-header',
   templateUrl: './comment-set-header.component.html',
   encapsulation: ViewEncapsulation.None
 })
-export class CommentSetHeaderComponent implements OnInit {
+export class CommentSetHeaderComponent implements OnInit, OnDestroy {
 
   @Input() public showCommentSummary: boolean;
   @Output() public readonly showCommentSummaryDialog = new EventEmitter();
 
-  tabs = [{label: 'comments'}, {label: 'filter'}, {label: 'search'}];
-  tabSelected: string;
+  tabs: {isFiltered?: boolean; label: string}[] = [];
+  tabSelected = 'comments';
   isFiltered: boolean;
   navigationList: Annotation[];
+  $ubscriptions: Subscription;
```

Comment:
I assume this is supposed to be subscriptions$

Reply:
oh dear

Reply:
no
$subscription 
observable$

i thought i misspell it and been looking like 1/2 minute where is the typo